### ISSUE TITLE
install-linux.sh can't find desktop folder on non-english systems

### ISSUE THREAD

**Issue by johannesgunnar** [tidalcycles/Tidal#71]
https://github.com/tidalcycles/Tidal/blob/master/doc/install-linux.sh

```
./install-linux.sh: lína 53: cd: /home/johannesg/Desktop: No such file or directory    
```

The install-linux.sh script fails on the last few lines (line 53) because of it trying to cd into the desktop. Normally this would work but on Linux distros with other languages the folders in the home directory often have a non-english name. In my case desktop is "Skjáborð". (Icelandic)

```
cd ~/Desktop
```

Isn't there some environment variable out there to refer to the Desktop directory, no matter its name?

**Comment by lvm** [comment]
Yeah, there's this tool http://www.freedesktop.org/wiki/Software/xdg-user-dirs/ (which in Debian based distros is called... `xdg-user-dirs`). You could find the default config here: `/etc/xdg/user-dirs.conf` or in your `$HOME/.config/user-dirs.dirs`.

TL;DR:  ` $ xdg-user-dir DESKTOP` should print your desktop directory in your language.

**Comment by fatmilktv** [comment]
Agreed valid; there's really no need for an Ubuntu-centric Desktop script.
There are other distributions but also, doesn't Tidal by now seem to prefer
Atom anyway? Where script is emacs. I'm behind on this.

Mauro are you suggesting a call against xdg-open to determine the proper
directory? Useful workaround I  imagine although if not installed it could
tank.

In the past I've considered making stat calls against the package manager
tree to see what's there, but surely there's a best practice for multiple
linuxes.

What about a start thing in local src plus an optional
append-to-local-path? /use/local/bin/Tidal or so .

**Comment by lvm** [comment]
Hi,
I already sent a PR with the necessary changes to make this workaround go smooth (ie: install a dependency and the command). Popular Desktop Environments (gnome, kde, xfce, mate, unity) already use this and seems to be the *standard* way to discover these directories. I mean, if you don't fall in this spectrum (don't use a popular DE or don't want to use xdg-user-dirs) probably you already know how to install Tidal by hand :-)

**Comment by fatmilktv** [comment]
Cool with me. I run Arch and I happen to not run a desktop but I can submit
all that as a distinct PR. And you're right.

**Comment by yaxu** [comment]
Lets focus thsi discussion on https://github.com/tidalcycles/Tidal/issues/284
